posts/2019-04-08-roadmap.md

```diff
@@ -0,0 +1,74 @@
+
+The work we have planned for this year falls into three major categories:
+
+* Governance: how the project is run
+* Finish long-standing requests: some important RFCs are not yet finished
+* Polish: improve the overall quality of the language and tooling
+
+As you can see, we still plan to do a lot this year, but it's more "cleanup work"
+than "make grand new plans."
+
+## Governance
+
+Over the last three years, the Rust project has grown a lot. Rust used to have a core team of 8 members. When we added sub-teams in 2015, we grew to 23 members. We've now grown to over 100 -- that's bigger than many companies! And of course, looking beyond the teams, the size of the Rust community as a whole has grown tremendously as well. As a result of this growth, we've found that the processes which served us well when we were a smaller project are starting to feel some strain.
+
+Many of the teams have announced plans to look over revamp their processes to scale better. Often this can be as simple as taking the time to write down things that previously were understood only informally -- sometimes it means establishing new structures. 
+
+Because of this widespread interest in governance, we've also created a new [**Governance Working Group**][gov]. This group is going to be devoted to working with each team to hone its governance structure and to help pass lessons and strategies between teams.
+
+[gov]: https://internals.rust-lang.org/t/governance-working-group-announcement/9637
+
+Additionally, the RFC process has been a great boon for Rust, but as we've grown, there have been times where it didn't work so well too.
+We may look at revisions to the process this year.
+
+## Long-standing requests
+
+Over the past few years, there have been a number of initiatives that were started but which, for one reason or another, never quite got finished. Many teams this year are taking a look at those long-standing items and trying to finish them out.
+
+Examples include:
+
+- The Cargo team and custom registries
+- The Language team is taking a look at async/await, specialization, const generics, and generic associated types
+- The Libs team wants to finish custom allocators
+
+A lot of "new" features in Rust this year will be this kind of thing: finishing features that we started long ago, but never
+got across the finish line.
+
+## Polish
+
+Finally, the last few years have also seen a lot of foundational work. The compiler, for example, was massively refactored to support incremental compilation and to be better prepared for IDEs. 
+
+Examples:
+
+- Compile times and IDE support
+- Polishing the specification of the language by improving [the reference] and laying out [the unsafe code guidelines]
+- The WebAssembly WG's work this year includes polishing our wasm support, for example, debugging
+
+[the reference]: https://doc.rust-lang.org/stable/reference/
+[the unsafe code guidelines]: https://github.com/rust-lang/unsafe-code-guidelines
+
```

Comment:
Maybe insert a "conclusion" here, something like

---

## Conclusion

This post only covered a few examples of the plans we've been making. [If you'd like to see the full details, take a look at the RFC itself.][rfc]